src/components/graphQLConnect.ts

```diff
@@ -70,6 +71,92 @@ class GraphQLdb {
     }
     // Not implemented yet -- needs more data in DB
   }
+
+  public getTemplateData = async (templateId: number): Promise<Template> => {
```

Comment:
Hmm, i am actually not sure about this, I think we've decided that dynamically displaying carbone template for the whole application is too difficult, and wanted to just provide a template that shows how to render each individual type of element (and their variations), and for carbone templates to be build for each type of application ?

I want to have a chat about this as I have an idea of how extra data can be provided to the generate carbone action

Reply:
> I think we've decided that dynamically displaying carbone template for the whole application is too difficult, and wanted to just provide a template that shows how to render each individual type of element (and their variations)

Yes, but will still need access to the question data from the template one way or another. So we can display something like (to user in doc):

> **Q1:** Company Name
> _Pharmac Importers_

Without the question data (labels, etc.), the responses alone won't be much use.

> I want to have a chat about this as I have an idea of how extra data can be provided to the generate carbone action

I'm happy for getting this extra data to be specific to the "generateDoc" action, I just thought it might be useful elsewhere too. We can have a chat in the morn.